modules/core/src/main/scala/org/scalasteward/core/nurture/NurtureAlg.scala

```diff
@@ -92,7 +92,7 @@ final class NurtureAlg[F[_]](config: ForgeCfg)(implicit
   private def processUpdate(data: UpdateData): F[ProcessResult] =
     for {
       _ <- logger.info(s"Process update ${data.update.show}")
-      head = forge.listingBranch(config.tpe, data.fork, data.updateBranch)
+      head = config.tpe.pullRequestHeadFor(data.fork, data.updateBranch)
```

Comment:
Note that Codecov is asserting [that line #L95 is not covered by tests](https://github.com/scala-steward-org/scala-steward/pull/3303/checks?check_run_id=22110826824), but `pullRequestHeadFor()` certainly is covered by tests (in `ForgeTypeTest`), and line 95 itself is not _added_, just changed - I feel this is a warning that can be ignored.